### ISSUE TITLE
go get

### ISSUE THREAD

**Issue by gregtampa** [gophish/gophish#1973]
**Thanks for reaching out! We're happy to help resolve issues as quickly as possible.**

**Please use this template when creating a new issue. If you do not follow this template format, your issue may be closed without comment.**

Before filing a new issue, please use the search bar at the top of the browser to search for similar issues. Also, please make sure you have read our [documentation](https://getgophish.com/documentation/), which covers how to use Gophish.

What version of Gophish are you using?: latest

Brief description of the issue: 
running command go get github.com/gophish/gophish
What are you expecting to see happen? :
a folder named gophish
What are you seeing happen?
```
go get github.com/gophish/gophish
# github.com/gophish/gophish/dialer
/root/go/src/github.com/gophish/gophish/dialer/dialer.go:78:10: unknown field 'Control' in struct literal of type net.Dialer
# github.com/emersion/go-message
/root/go/src/github.com/emersion/go-message/charset.go:24:9: undefined: errors.As
/root/go/src/github.com/emersion/go-message/encoding.go:27:9: undefined: errors.As
```
If this question is related to email templates or landing pages not working as expected, please provide your template or landing page below:

```html
Insert code here
```

Please provide any terminal output that may be relevant below:

```
root@gophish:/opt/greg# go get github.com/gophish/gophish

# github.com/gophish/gophish/dialer
/root/go/src/github.com/gophish/gophish/dialer/dialer.go:78:10: unknown field 'C                                                                                                            ontrol' in struct literal of type net.Dialer
# github.com/emersion/go-message
/root/go/src/github.com/emersion/go-message/charset.go:24:9: undefined: errors.A                                                                                                            s
/root/go/src/github.com/emersion/go-message/encoding.go:27:9: undefined: errors.                                                                                                            As
root@gophish:/opt/greg#

```

Please provide as many steps as you can to reproduce the problem:
 * Step 1 - Ubuntu 18.04.5 LTS
 * Step 2 - go version go1.10.4 linux/amd64

**Comment by glennzw** [comment]
Your Go version is just over two years old; might be worth trying with a newer version.

**Comment by jordan-wright** [comment]
@glennzw is correct- I recommend updating your version of Go. I'll update the README soon with the next lowest version supported.

**Comment by etnoy** [comment]
Just a heads up, this will happen when using the goland version included in Ubuntu 18.04 LTS which is still supported.

You can use a PPA to get it to work:

sudo apt-get install software-properties-common gpg
sudo add-apt-repository ppa:longsleep/golang-backports
sudo apt-get update
sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 52B59B1571A79DBC054901C0F6BC817356A3D45E
sudo apt-get install golang-go
